### ISSUE TITLE
[BUG] Incompatibility with Lenient Death

### ISSUE THREAD

**Issue by neonjuno** [RedPxnda/RespawnObelisks#24]
**Describe the bug**
Using the keep inventory feature from Respawn Obelisks while having Lenient Death mod installed makes items disappear when preserveItemsOnDeath from Lenient Death is disabled.

**To Reproduce**
1. Turn on any keep inventory setting in Respawn Obelisks
2. Turn off preserveItemsOnDeath in Lenient Death
3. Die
4. Spawn back with an empty inventory
Items that were set to be kept by RO are deleted from the world, the rest of the items are dropped.

**Expected behavior**
I expect to keep all the items that were deleted.

**Versions**
- Minecraft Version 1.20.1
 - Respawn Obelisk Version 2.6-b
 - Fabric Loader Version 0.16.9
 - Fabric API Version 0.92.2
 - Lenient Death Version 1.2.5

**Additional notes**
I've opened an issue on their page since I think it's a problem on their end but I thought I'd open one here too for the record.

**Comment by RedPxnda** [comment]
Just by the sound of it, I agree with you that it sounds like a problem on their end. Although I could be completely wrong about that.
I'd had a quick look now and didn't find much, I'll have to have a closer look later but unfortunately I have a lot on my hands right now. I might wait to see what the dev of Lenient Death has to say.

**Comment by neonjuno** [comment]
Yeah I completely understand, I love this mod and how customizable it is so thanks for all the work
